src/main/app/src/app/informatie-objecten/informatie-object-edit/informatie-object-edit.component.ts

```diff
@@ -110,7 +110,7 @@ export class InformatieObjectEditComponent implements OnInit, OnDestroy {
       .build();
 
     const taal = new SelectFormFieldBuilder({
-      naam: this.translateService.instant(this.infoObject.taal.naam),
+      naam: this.translateService.instant(this.infoObject.taal?.naam),
```

Comment:
Now what happens when taal is null? Should the null check not be done before even setting the `naam` here?

Reply:
I've changed it a bit and made sure it looks correct in the UI as well. This happens when the language field in OpenZaak is something we can't map to a known value. In that case, taal is null. This can happen if you do a productaanvraag. This is what it looks like if you try to add a new version:
![image](https://github.com/infonl/dimpact-zaakafhandelcomponent/assets/104420825/85358da8-62bd-4048-8253-290eb86edfc1)